modules/ichorcna/1.1/config/default.yaml

```diff
@@ -24,19 +24,40 @@ lcr-modules:
                 ichorCNA_normalPanel: 
                     "1000000": "inst/extdata/HD_ULP_PoN_{genome_build}_1Mb_median_normAutosome_median.rds"
                     "500000": "inst/extdata/HD_ULP_PoN_{genome_build}_500kb_median_normAutosome_median.rds"
+                # relative to ichorCNA directory (in 00-inputs/ichorCNA/)
```

Comment:
Can you provide some documentation about when/why someone might want to use custom normalPanel and gcWig files?